README.md

```diff
@@ -27,6 +27,11 @@ Demonstrate calling the method with an argument of "young prince".
 Write your code here:
 ```ruby
 # code here
+def offerRose(person)
+  @person = person
```

Comment:
No need for an instance variable here, unless you wanted to use this person in another plot line.  :)